arcor2_AREditor/Assets/TABLET/Scripts/GUI/WalktroughHandler.cs

```diff
@@ -0,0 +1,252 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using Michsky.UI.ModernUIPack;
+using UnityEngine.UI;
+
+public class WalktroughHandler : MonoBehaviour {
```

Comment:
Use `OnValidate()` to check that all required GameObjects, etc. are set (not `null`). I think it is a good approach to prevent problems :)